tests/contrib/aredis/test_aredis.py

```diff
@@ -0,0 +1,241 @@
+# -*- coding: utf-8 -*-
+import aredis
+import pytest
+
+from ddtrace import Pin
+from ddtrace.constants import ANALYTICS_SAMPLE_RATE_KEY
+from ddtrace.contrib.aredis.patch import patch
+from ddtrace.contrib.aredis.patch import unpatch
+from tests.opentracer.utils import init_tracer
+from tests.utils import DummyTracer
+from tests.utils import assert_is_measured
+from tests.utils import override_config
+
+from ..config import REDIS_CONFIG
+
+
+@pytest.fixture
+async def traced_aredis():
```

Comment:
I know the existing redis integration does testing this way, but I wonder if we should just set these up now as snapshot tests?


Example form `rq`:

https://github.com/DataDog/dd-trace-py/blob/550957f8a0e33df1e6e63cfaa2ee08b8ea0ae4d5/tests/contrib/rq/test_rq.py#L98-L104

or example from `httpx`:

https://github.com/DataDog/dd-trace-py/blob/550957f8a0e33df1e6e63cfaa2ee08b8ea0ae4d5/tests/contrib/httpx/test_httpx.py#L50-L60

Makes for a much nicer test IMHO.

We can do this as a follow-up, but could be a nice addition to do now too.

Reply:
I pick up the httpx example.